101-cognitive-services-translate/azuredeploy.parameters.json

```diff
@@ -2,5 +2,14 @@
   "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+      "accountName": {
+          "value": "TextTranslation"
+      },
+      "SKU": {
+          "value": "F0"
+      },
+      "location": {
+          "value": "global"
```

Comment:
@bmoore-msft -  Microsoft.CognitiveServices/accounts needs location as 'global' to pass public deployment. Since there's only 1 resource in the template, not using separate location parameter.